src/Compiler/TaskTableView.cpp

```diff
@@ -263,13 +263,26 @@ void TaskTableView::TasksDelegate::paint(QPainter *painter,
     if (statusData.type() == QVariant::Bool) {
       label->setMovie(m_inProgressMovie);
       // Place the animation to cells left side, similar to other decorations
-      label->move(m_view.visualRect(index).topLeft());
+      label->move(m_view.visualRect(index).center() -
+                  (label->rect().center() - QPoint{6, 0}));
       m_inProgressMovie->start();
       return;
     } else {
       // Reset the movie when task is no longer in progress
       label->setMovie(nullptr);
     }
+
+    QStyleOptionViewItem opt = option;
```

Comment:
What is this code trying to do ?

Reply:
Create new object `opt` and copy all data from object `option`. Copy constructor is used here.

Reply:
For alignment, I see the code above but didn't follow why we need this change.

Reply:
This code need for alignment. What exactly is not clear?